dotnet/Devolutions.NowProto/src/Messages/NowMsgExecBatch.cs

```diff
@@ -73,26 +82,34 @@ public Builder Directory(string directory)
                 _directory = directory;
                 return this;
             }
+            public Builder IoRedirection()
+            {
```

Comment:
style: spacing

Reply:
huh, strange that dotnet not detected this